src/sage/groups/generic.py

```diff
@@ -1407,6 +1409,99 @@ def order_from_bounds(P, bounds, d=None, operation='+',
 
     return order_from_multiple(P, m, operation=operation, check=False)
 
+def has_order(P, n, operation='+'):
+    r"""
+    Generic function to test if a group element `P` has order
+    exactly equal to a given positive integer `n`.
+
+    INPUT:
+
+    - `P` -- group element with respect to the specified ``operation``
+    - `n` -- positive integer, or its :class:`~sage.structure.factorization.Factorization`
+    - ``operation`` -- string, either ``'+'`` (default) or ``'*'``
```

Comment:
```suggestion
    - ``P`` -- group element with respect to the specified ``operation``
    - ``n`` -- positive integer, or its :class:`~sage.structure.factorization.Factorization`
    - ``operation`` -- string, either ``'+'`` (default) or ``'*'``
```
We always make these code formatted.